examples/skr/aci/README.md

```diff
@@ -36,7 +38,7 @@ The managed HSM instance endpoint corresponds to [`SkrClientAKVEndpoint`](aci-ar
 
 If you don't already have a valid attestation endpoint, create a [Microsoft Azure Attestation](https://learn.microsoft.com/en-us/azure/attestation/overview) endpoint to author the attestation token and run the following command to get the endpoint value:
 
-```
+```shell
 az attestation show --name "<ATTESTATION PROVIDER NAME>" --resource-group "<RESOURCE GROUP>"

```

Comment:
Bryce wanted us to remove this entire section where we do az attestation show command. Instead, we should just include a maa endpoint on the sample yaml or arm template.

Reply:
When did he say that? Removing this gets rid of steps that might be useful to customers.

Reply:
We're really only publishing the published endpoints.  Most customers will never make their own MAA endpoint.